### ISSUE TITLE
Clicking on repo name causes repo quick switch button to disappear

### ISSUE THREAD

**Issue by adamreisnz** [ZenHubIO/support#1174]
Observe when you have the repo switcher visible:

![image](https://user-images.githubusercontent.com/490562/27940490-14877466-631f-11e7-876a-a8b2263dd32d.png)

Click on the name of the repo, and the button disappears and doesn't come back:

![image](https://user-images.githubusercontent.com/490562/27940507-2118ca4a-631f-11e7-98cd-6fbeec9343e5.png)

Annoying as I often click on the name to reload the page go back to the main page.

**Comment by kendramoroz** [comment]
@adamreisnz 🤔  that's not intended! It should come back upon refresh of the page. Is there any errors you're noticing in the console when it disappears?

**Comment by adamreisnz** [comment]
Nope, no errors in the console. It *does* come back after a hard browser refresh, but not when clicking on the link to "soft refresh" the page.

I noticed today when I tried it again, that the first time I clicked it did come back, but then the second and subsequent times it didn't anymore.

Have you tried to reproduce it yourself as well?

**Comment by kendramoroz** [comment]
>  that the first time I clicked it did come back, but then the second and subsequent times it didn't anymore.

ah ha! That was what I was missing in my debugging. I was able to reproduce this with the second click. Bumping over to known issues. Thanks for the note on that!

**Comment by adamreisnz** [comment]
Cool thank you!
